src/index.test.ts

```diff
@@ -172,9 +172,23 @@ test('Should include context fields on request', async () => {
         [JSON.stringify(data), { status: 200 }],
         [JSON.stringify(data), { status: 304 }],
     );
-    const config: IConfig = { url: 'http://localhost/test', clientKey: '12', appName: 'web', environment: 'prod' };
+    const context: IMutableContext = {
+        userId: '123',
+        sessionId: '456',
+        remoteAddress: 'address',
```

Comment:
Can we add hostName to the context?

Reply:
Would that be the hostName of the end-user? Or the service hostName?

Reply:
The hostName of the end-user.
The use case is to send "window.location.hostname" as the domain of the used webapp as a parameter. This can enable staging environments and dev environments to have experimental features toggled

I see that remoteAddress can be used for this. 

I do not know what the unleash community thinks about this. As it seems it has not been thought of.